lib/gitsh/environment.rb

```diff
@@ -88,7 +87,7 @@ def repo_has_untracked_files?
     end
 
     def repo_config_color(name, default)
-      if color_override = self[name]
+      if color_override = fetch(name) { nil }
```

Comment:
I removed the `#[]` method, to avoid having a method that might return `nil`: This is one of the very few cases where a `nil` would be useful, and I thought it was better to be explicit and use the slightly odd looking `{ nil }` here, rather than keep the `#[]` method that might let `nil` values accidentally leak into places where they're not wanted.